localflavor/au/validators.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.core.exceptions import ValidationError
+from django.core.validators import RegexValidator
+from django.utils.translation import ugettext_lazy as _
+
+
+class AUBusinessNumberFieldValidator(RegexValidator):
+    """
+    Validation for Australian Business Numbers.
+
+    .. versionadded:: 1.3
+    """
+
+    error_message = _('Enter a valid ABN.')
+
+    def __init__(self):
+        eleven_digits_with_whitespace = '^(\s*\d\s*){11}$'
```

Comment:
This should be updated to only check for digits since the spaces can be removed elsewhere.